tests/unit/src/test/scala/tests/debug/BreakpointScalaCliDapSuite.scala

```diff
@@ -0,0 +1,42 @@
+package tests.debug
+
+import tests.{BaseDapSuite, QuickBuildInitializer, ScalaCliBuildLayout}
+
+class BreakpointScalaCliDapSuite
+    extends BaseDapSuite(
+      "scala-cli-debug-breakpoint",
+      QuickBuildInitializer,
+      ScalaCliBuildLayout,
+    ) {
+
+  private val scalaCliScriptPath = "a/src/main/scala/a/script.sc"
+  assertBreakpoints(
+    "script",
+    main = Some("a.src.main.scala.a.script_sc"),
+    buildTarget = Some("project_28a1ba13eb"),
```

Comment:
This will probably change, so I will take a look at it later to have a stable name.